chain/network/src/peer.rs

```diff
@@ -495,7 +495,7 @@ impl StreamHandler<Vec<u8>, io::Error> for Peer {
                     actix::spawn(
                         self.peer_manager_addr
                             .send(NetworkRequests::BanPeer {
-                                peer_id: self.node_info.id.clone(),
+                                peer_id: self.peer_info.id.clone(),
```

Comment:
As far as I understand it should ban a node which sent the message, right? Like `routed_message.author"?

Reply:
@fckt this implementation will change soon, anyway, in this case the one that should be banned is the immediate sender of the message not the author. Rationale about this is:
- Intermediate hops can alter the message.
- Previous hop must check signature before passing the message. 
The only who can ban the author is the final receiver of the message in case the content is invalid. Intermediate hops only check integrity of the message, i.e. hash of the content and signature match the sender's public key.